### PR TITLE
Update copyright year

### PR DESCRIPTION
**Pull request by wm4**

### CODE REVIEW

**Comment by rossy** [comment]
This too?

https://github.com/mpv-player/mpv/blob/7677c7c32c3c/osdep/macosx_menubar.m#L615

**Comment by wm4** [comment]
That one seems new. Looks like the text should just be moved to some define or so to share it.

Could this be done with mpv.rc too?

**Comment by rossy** [comment]
> Could this be done with mpv.rc too?

I think so. Resource files use a C-ish preprocessor so it should understand simple preprocessor defines. The Unicode © is fine as well, I think.

**Comment by wm4** [comment]
Made an attempt, but didn't test on win32 or OSX.

CC @Akemi

**Comment by Akemi** [comment]
menubar and the About menu still work as intended.

File: osdep/mpv.rc [hunk 1]
```diff
@@ -16,6 +16,7 @@
  */
 
 #include <winver.h>
+#include "version.h"
```
Comment (rossy) [inline osdep/mpv.rc]: Re-adding "version.h" to mpc.rc brings back the build failures from 3375d65bc5c1. For some reason we couldn't figure out how make waf wait for version.h to be generated before compiling mpc.rc.

If we can figure out how to fix this, we could also go back to having real version numbers in mpc.rc.

**Comment by rossy** [comment]
mpv.rc is good and the Unicode © works, but the version.h thing is an issue (see comments.)

**Comment by wiiaboo** [comment]
> the Unicode © works
In file properties it shows up as broken image here.

@wm4 did you want to add VERSION to mpv.rc as well?

**Comment by wm4** [comment]
Not necessarily, but feel free to add them if it makes sense,

The (C) sign thing is probably because the resource compiler doesn't understand utf-8?

**Comment by wiiaboo** [comment]
Yeah, had to add `--codepage=65001` to WINDRES_FLAGS and it works now.

**Pull request closed without merging**